src/main/scala/xiangshan/frontend/NewFtq.scala

```diff
@@ -810,6 +822,8 @@ class Ftq(implicit p: Parameters) extends XSModule with HasCircularQueuePtrHelpe
   io.toICache.req.valid := entry_is_to_send && ifuPtr =/= bpuPtr
   io.toICache.req.bits.readValid.zipWithIndex.map{case(copy, i) => copy := toICacheEntryToSend(i) && copied_ifu_ptr(i) =/= copied_bpu_ptr(i)}
   io.toICache.req.bits.pcMemRead.zipWithIndex.map{case(copy,i) => copy.fromFtqPcBundle(toICachePcBundle(i))}
+  io.toICache.req.bits.backendIpf := backendIpf
+  io.toICache.req.bits.backendIaf := backendIaf
```

Comment:
Just to leave a memo here: in new ICache, a fetch request is first sent to IPrefetch (instead of ICacheMainPipe), and the iTLB query is done there. So in new ICache, these two signals should also be sent to IPrefetch and enqueued to WayLookup along with the iTLB result.

If this PR is merged to master before new ICache, we need to keep an eye on this while resolving merge conflicts.